### PR TITLE
completed traits list for listing-body 

### PR DESCRIPTION
**Pull request by elimoreh**

### CODE REVIEW

File: Module/client/src/components/app.jsx [hunk 1]
```diff
@@ -17,25 +18,49 @@ class App extends React.Component {
     this.getData();
   }
 
+  // initial and only request to server for mockData
   getData() {
     axios.get('/listinginfo').then((response) => {
       let { id } = this.state;
-      if (!id || (id > 100) || (id < 0)) { id = 0; }
+      // sets id to the first mock data example if id is undefined or outside range
+      if (!id || (id > 101) || (id < 0)) { id = 0; }
```
Comment (sbtorres) [inline Module/client/src/components/app.jsx]: I get the idea behind doing this, but wouldn't it make more sense for it to show a 404 page not found?

File: Module/client/src/components/app.jsx [hunk 2]
```diff
@@ -17,25 +18,49 @@ class App extends React.Component {
...
   render() {
     const { loading, data } = this.state;
     if (loading) return null;
-    const { hostName, header, neighborhood } = data;
+    // deconstructing data that will be sent down as props
+    const {
+      hostName, homeType, header, neighborhood, bathrooms, bedrooms, capacity,
+      beds, primTrait, secondTrait, tertTrait,
+    } = data;
+    // similar to Airbnb, this sets the description as tab title
```
Comment (sbtorres) [inline Module/client/src/components/app.jsx]: This is well done, I hadn't noticed that they do that.

File: Module/client/src/components/body.jsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+// styled components that will be used as classes for styling
+import {
+  BodyContainer, TraitBox, LogoBox, Icon, TraitDesc, TraitHeader, HomeDetails, Detail,
+} from '../styles/style';
+
+const Body = (props) => {
+  const {
+    hostName, homeType, bathrooms, bedrooms, beds, capacity, primTrait, secondTrait, tertTrait,
+  } = props;
+  // creates one unifed logo from a specific array of svg paths declared in the styles/svg file
```
Comment (sbtorres) [inline Module/client/src/components/body.jsx]:
```suggestion
  // creates one unified logo from a specific array of svg paths declared in the styles/svg file
```
small typo.

File: Module/client/src/styles/style.js [hunk 4]
```diff
@@ -91,14 +92,86 @@ const HostImageBox = styled.img`
...
+    width: 100% !important;
+    border-spacing: 0px !important;
+  `;
+const LogoBox = styled.div`
+    display: table-cell !important;
+    vertical-align: top !important;
+  `;
+
+const Icon = styled.svg`
+  position:absolute;
```
Comment (sbtorres) [inline Module/client/src/styles/style.js]:
```suggestion
  position: absolute;
```
small typo

File: Module/client/src/styles/style.js [hunk 5]
```diff
@@ -91,14 +92,86 @@ const HostImageBox = styled.img`
...
+  display: block;
+  fill: #484848;
+  width: 16px;
+  length:20px;
```
Comment (sbtorres) [inline Module/client/src/styles/style.js]:
```suggestion
  length: 20px;
```

File: Module/client/src/styles/svg.jsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```
Comment (sbtorres) [inline Module/client/src/styles/svg.jsx]: I really like that you separated this out into its' own file.

**Review by sbtorres — CHANGES_REQUESTED** [review]
Some small typos you probably want to clean up before merging. Also let me know what you think about the 404 page found idea. May not make sense for this project... but I think it's more realistic.

**Review by michelledhuang23 — APPROVED** [review]
it looks awesome!! I don't have any other suggestions XD

File: Module/client/src/components/app.jsx (hunk 1 above)
Comment (elimoreh) [reply Module/client/src/components/app.jsx]: Yeah when in production i'll make it send a 404, but for now sometimes i forget to type in the id number. Thanks!

File: Module/client/src/components/app.jsx (hunk 2 above)
Comment (elimoreh) [reply Module/client/src/components/app.jsx]: Thanks!

**Comment by elimoreh** [comment]
Thanks guys!! @sbtorres good catches on those typos and i'll for sure change to 404 before i finalize.

**Review by sbtorres — APPROVED** [review]

**Pull request merged**